app/helpers/app_text_helper.rb

```diff
@@ -15,6 +15,17 @@ def label_text(key, options = {})
     named_text(:labels, key, options)
   end
 
+  def label_text_with_subject(key, subject, options = {}, subject_appended: true)
```

Comment:
I think the label text helper does handle subject. Why is this necessary?

Reply:
It does, but I needed a bit more flexibility to cover scenarios that deal with subjects. For many of the summary boxes we want to display the subject, but only if a partner exists. So I decided to be explicit about it with this method. I suppose in most cases I could've added two versions of each title to the locales, e.g. "Payments" and "Payments: %{subject}", but I thought just having one and appending the subject there might be nicer?

There's also one case where the wording is different (plural) if we're dealing with a client and a partner: `_other_income_details.html.erb:6` and `casework.yml:261`, so I wanted to account for that in the same method as well.

Reply:
I see, thanks!